app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Api
+  module V1
+      class UsersController < ApplicationController
+        skip_before_action :doorkeeper_authorize!, only: %i[create]
+    
+        def create
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/blyscuit/rails-example-one/blob/ffe479ef97670c7333f37452d55369f226bbc764/app/controllers/api/v1/users_controller.rb#L6"></span>Assignment Branch Condition size for create is too high. [&lt;3, 29, 3&gt; 29.31/17]</td>
    </tr>
  </tbody>
</table>